packages/twenty-front/src/modules/object-record/advanced-filter/components/AdvancedFilterValueInputDropdownButton.tsx

```diff
@@ -79,14 +78,12 @@ export const AdvancedFilterValueInputDropdownButton = ({
             setSelectedFilter(filter);
           }}
           dropdownComponents={
-            <DropdownMenuItemsContainer>
-              <ObjectFilterDropdownFilterInput />
-            </DropdownMenuItemsContainer>
+            <ObjectFilterDropdownFilterInput recordFilterId={filter.id} />
           }
           dropdownHotkeyScope={{ scope: dropdownId }}
           dropdownOffset={{ y: 8, x: 0 }}
           dropdownPlacement="bottom-start"
-          dropdownMenuWidth={280}
+          dropdownMenuWidth={200}
```

Comment:
yes, but let's make sure the dropdownWidth is correct for the datetime picker (there is a setDropdownWIdth function)